From 66c4b6a16af01b3d82dc606ef728887fa2a79d4b Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Thu, 14 Jul 2011 20:07:14 +0800
Subject: ENGR00152371-3 imx6_defconfig: enable vpu config

Enable CONFIG_IMX_HAVE_PLATFORM_IMX_VPU and CONFIG_MXC_VPU
for imx6q.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index e810c81c..11955f8 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux/arm 2.6.38 Kernel Configuration
-# Wed Jul 13 14:27:14 2011
+# Thu Jul 14 19:45:47 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -260,6 +260,7 @@ CONFIG_IMX_HAVE_PLATFORM_MXC_PWM=y
 CONFIG_IMX_HAVE_PLATFORM_SDHCI_ESDHC_IMX=y
 CONFIG_IMX_HAVE_PLATFORM_SPI_IMX=y
 CONFIG_IMX_HAVE_PLATFORM_IMX_IPUV3=y
+CONFIG_IMX_HAVE_PLATFORM_IMX_VPU=y
 CONFIG_IMX_HAVE_PLATFORM_AHCI=y
 CONFIG_IMX_HAVE_PLATFORM_LDB=y
 CONFIG_IMX_HAVE_PLATFORM_VIV_GPU=y
@@ -1662,6 +1663,8 @@ CONFIG_MXC_IPU_V3H=y
 #
 # MXC VPU(Video Processing Unit) support
 #
+CONFIG_MXC_VPU=y
+# CONFIG_MXC_VPU_DEBUG is not set
 
 #
 # MXC Asynchronous Sample Rate Converter support
-- 
1.7.9.5

